app/assets/stylesheets/application.css

```diff
@@ -1,15 +1,415 @@
+  margin-right: 5px;
+}
+
+form .links {
+  text-align: center;
+}
+
+.footer-links {
+  text-align: center;
+  margin-top: 20px;
+}
+
+.footer-links a {
+  text-decoration: none;
+  color: blueviolet;
+}
+
+.footer-links a:hover {
+  text-decoration: underline;
+}
+
+.categories-container {
+  padding: 10px;
+}
+
+.categories-container .header {
+  background-color: #007bff;
+  color: white;
+  text-align: center;
+  margin: 0;
+}
+
+.category-item {
+  background-color: #fff;
+  border: 1px solid #ddd;
+  border-radius: 5px;
+  padding: 10px;
+}
+
+.category-card {
+  display: flex;
+  align-items: center;
+  gap: 1em;
+}
+
+.category-image {
+  flex-shrink: 0;
+}
+
+.category-details {
+  display: flex;
+}
+
+.category-name {
+  color: #000;
+  text-decoration: none;
+  font-size: medium;
+  font-weight: bold;
+  margin: 0;
+}
+
+.total-amount {
+  display: flex;
+  justify-content: space-between;
+  color: black;
+  text-align: left;
+  font-weight: bold;
+  margin-top: 28px;
+  order: 1;
+}
+
+.category-list {
+  list-style-type: none;
+  padding: 0;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-gap: 20px;
+}
+
+h1 {
+  font-size: 24px;
+  margin-bottom: 20px;
+}
+
+.label {
+  font-weight: bold;
+}
+
+#icon-select {
+  width: 100%;
+  padding: 10px;
+}
+
+#selected_icon {
+  margin-top: 10px;
+}
+
+/* transactions.css */
+.transactions-container {
+  background-color: #fff;
+  padding: 10px;
+  width: 100%;
+  border: 1px solid #fff;
+  border-radius: 5px;
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.transactions-container ul {
+  display: flex;
+  flex-direction: column;
+  order: 2;
+}
+
+.category-header {
+  display: flex;
+}
+
+.category-header1 img {
+  max-width: 100px;
+  max-height: 100px;
+  margin-right: 10px;
+}
+
+.transaction-item {
+  border-top: 1px solid #ccc;
+  padding-top: 20px;
+  margin-top: 10px;
+  list-style: none;
+  border-bottom: 1px solid #ccc;
+  padding-bottom: 20px;
+}
+
+.amount {
+  font-weight: bold;
+  margin-top: 15px;
+}
+
+/* transactions.css */
+.transaction-form-container {
+  background-color: #fff;
+  padding: 20px;
+  border: 1px solid #ddd;
+  border-radius: 5px;
+  margin-top: 20px;
+}
+
+.new-category {
+  display: inline-block;
+  padding: 15px 0;
+  background-color: #5fb523;
+  color: white;
+  border-radius: 0;
+  font-weight: bold;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+  width: 100%;
+  text-align: center;
+  position: fixed;
+  bottom: 0;
+}
+
+a.new-category-button {
+  background-color: #5fb523;
+  width: 80%;
+  display: inline-block;
+  border: 1px solid #ddd;
+  text-decoration: none;
+  text-align: center;
+  color: #fff;
+  padding: 15px 0;
+  border-radius: 5px;
+}
+
+a.new-category-button:hover {
+  background-color: #4a8e18;
+}
+
+div.new-category-button-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #5fb523;
+}
```

Comment:
- [ ] As noted by the previous reviewer [here](https://github.com/BenMKT/Budget-App/pull/1#:~:text=After%20creating%20a%20transaction%20for%20the%20first%20time%20the%20button%20doesn%27t%20show%20well.%20It%20only%20shows%20better%20after%20reloading%20the%20page.%20Check%20the%20video%20below), the button fails to retain the intended style after the creation of a transaction 🐛However, it displays correctly in accordance with the design specifications when redirected from the home page.
- [ ] As mentioned earlier, please ensure that the button spans a width similar to the provided design⛏🤓 _Please have a look at the attached video 📹 You can fix this by adding some_

https://github.com/BenMKT/Budget-App/assets/106119105/109278e6-6eae-4cb4-9c53-d5a530171bc6

Reply:
- Added `padding` and `height` to `div.new-category-button-container` to resolve bug.

Reply:
This change has been done well. The button now spans the entire width